Base ДЗ 1

tUnknownLegend opened the pull request:
Колесников Артем
Максимова Елизавета
Пиневич Владислав
Федоров Александр

tUnknownLegend committed:
Create Base.md

tUnknownLegend committed:
Update Base.md

tUnknownLegend committed:
Item page, comments, header, footer, main page (#2)

* Страница товара

* Update Base.md

* Update Base.md

* Update Base.md

* Update Base.md

* imgs

* img

* Update Base.md

* img

* deleted imgs

* главная + футер

* Update Base.md

* Update Base.md

* Иправление замечаний

* Update Base.md

* Update Base.md

* Update Base.md

* Update Base.md

* исправление замечаний v2

* вынес разбите цены по тысячам в общие паттерны

* Update Base.md

thelvv committed:
Profile (#3)

* initial

* Сделал блок личной информации

* Доделал блоки платежных карт и карт адресов

* небольшой фикс

* фикс верстки

* добавил корректное поведение для багов

* исправил ошибки

* исправил ошибки

---------

Co-authored-by: Vlad Pinevich <57019979+tUnknownLegend@users.noreply.github.com>

Liza1040 committed:
Basket (#1)

* initial commit

* Протестирована плашка "Корзина"

* Тестовое фрматирование картинки

* Фикс ошибки

* Фикс 2

* Фикс 3

* Фикс 4

* Фикс 5

* Проверка

* Проверка 2

* Проверка 3

* Форматирование картинок

* Тест заголовка

* Тестирование заголовков

* Тест заголовка

* Протестированы все блоки страницы

* Добавлен перенос строки в конце файла

* Добавлен баг с обновлением страницы

* Добавлена ссылка на страницу в название

* Добавлено корректное поведение к багам и классификация багов

* Добавлены скрины каждого блока

* Поправлены скрины блоков

* Update Base.md

---------

Co-authored-by: Vlad Pinevich <57019979+tUnknownLegend@users.noreply.github.com>

EuphoriaAbsorber committed:
Favorite (#4)

* Create Base.md

* favorites

* orders page

* authorization

* category

* search page + pr comments fixes

* fixes

* pr comments fixes

* added links in contents

---------

Co-authored-by: Vlad Pinevich <57019979+tUnknownLegend@users.noreply.github.com>

tUnknownLegend committed:
Create .gitignore

tUnknownLegend committed:
Create README.md

tUnknownLegend commented:
Аппрув

EuphoriaAbsorber commented:
подтверждаю

Liza1040 commented:
Апрув

thelvv commented:
Аппрув

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+13. [Корзина](https://github.com/tUnknownLegend/TP-QA-HW-1/blob/favorite/Base.md#%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B0-%D0%BA%D0%BE%D1%80%D0%B7%D0%B8%D0%BD%D1%8B)
+
+# Общие паттерны
+- Кнопки и ссылки, покрашенные в цвет акцента становятся более темного цвета при наведении
```
Уточните что такое `цвет акцента` и что такое `более темного цвета`?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+
+# Общие паттерны
+- Кнопки и ссылки, покрашенные в цвет акцента становятся более темного цвета при наведении
+- Сообщения об ошибке появляются в правом верхнем углу, на красном фоне, исчезают через 3 секунды
```
Можно добавить скрин

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+- Кнопки и ссылки, покрашенные в цвет акцента становятся более темного цвета при наведении
+- Сообщения об ошибке появляются в правом верхнем углу, на красном фоне, исчезают через 3 секунды
+- Уведомления о произведенных действиях полностью аналогичны сообщениям об ошибке, но имеют белый фон и тень
+- Цена разделена по тысячам пробелом, между цифрами и валютой есть пробел, некликабельна
```
можно добавить пример

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+- При отсутствии заказов пользователю предлагается перейти в одну из категорий.
+### Карточка заказа
+![image](https://user-images.githubusercontent.com/65418582/222961384-966bf8c5-6dc9-4210-a82b-46c2022fbfe2.png)
+ - Номер заказа не совпадает с номерами других заказов пользователя.
```
Как это проверить?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+ - При клике на название товара загружается страница этого товара.
+ - Итоговая стоимость заказа совпадает с суммарной стоимостью всех товаров заказа.
+ - Параметры заказа: оплата, адрес, дата доставки, время доставки совпадают с указанными при создании заказа.
+ - Дата оформления заказа проставляется корректно.
```
Что есть корректно?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+ - Параметры заказа: оплата, адрес, дата доставки, время доставки совпадают с указанными при создании заказа.
+ - Дата оформления заказа проставляется корректно.
+ - При нажатии на кнопку "Отменить заказ" статус заказа меняется с "Создан" на "Отменен", пропадает кнопка отмены.
+ - *Недоработка*. Одинаковый тип оплаты при создании заказа с привязанной картой и при создании заказа с выбором оплаты на месте.
```
Степень влияния бага вам нужно будет оценить во второй домашке, в этой просто пишете всегда БАГ

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+- При отсутствии товаров в избранном пользователю предлагается перейти в одну из категорий.
+- *Значительный баг*. Список товаров в избранном без применения сортировок некорректен, некоторые товары дублируются.
+  - *Корректное поведение*. Ровно по одной карточке каждого товара.
+- Сортировка по рейтингу по убыванию и возрастанию работает корректно.
```
Опять же от корректно лучше избавить в данном случае и стройкой ниже проще переформулировать

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+  - *Корректное поведение*. Ровно по одной карточке каждого товара.
+- Сортировка по рейтингу по убыванию и возрастанию работает корректно.
+- Сортировка по цене по убыванию и возрастанию работает корректно.
+- Иконка направления сортировки отображается корректно.
```
Опять же что значит корректно здесь?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+ - При клике на название загружается страница этого же товара.
+ - При клике на фотографию загружается страница этого же товара.
+ - Кнопка "В корзину"
+     - Корректно отображает количество товаров в корзине
```
Опять же корректно

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+ - При клике на фотографию загружается страница этого же товара.
+ - Кнопка "В корзину"
+     - Корректно отображает количество товаров в корзине
+     - Можно добавить товар в корзину, счетчик увеличится.
```
Можно -> если

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+ - Кнопка "В корзину"
+     - Корректно отображает количество товаров в корзине
+     - Можно добавить товар в корзину, счетчик увеличится.
+     - Можно удалить товар из корзины, счетчик уменьшится.
```
Аналогично комменту выше

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+     - Корректно отображает количество товаров в корзине
+     - Можно добавить товар в корзину, счетчик увеличится.
+     - Можно удалить товар из корзины, счетчик уменьшится.
+     - Если удалить последний экземпляр товара, то кнопка "В корзину" примет стандартный внешний вид.
```
Что такое `стандартный внешний вид`?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+- Длинное название товара помещается полностью.
+- При клике на название или на фотографию загружается страница этого же товара.
+- Рейтинг и кнопка "В корзину" работают аналогично таковым на карточке товара.
+- Если товар со скидкой, то процент скидки указывается корректно.
```
Что есть корректно?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+# [Категории](https://www.reazon.ru/category/phones)
+### Общее
+- Все товары на странице категории относятся к этой категории.
+- Сортировка по рейтингу по убыванию и возрастанию работает корректно.
```
Что такое `эта` категория?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+### Общее
+- Все товары на странице категории относятся к этой категории.
+- Сортировка по рейтингу по убыванию и возрастанию работает корректно.
+- Сортировка по цене по убыванию и возрастанию работает корректно.
```
Опять же `корректно`, это антипаттерн, посмотрите по всему чек листу что у вас нет такой фразу

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+- Сортировка по цене по убыванию и возрастанию работает корректно.
+- Иконка направления сортировки отображается корректно.
+### Карточка товара
+Аналогично карточке товара из раздела "Избранное".
```
Можно прям ссылку в чеклисте добавить, будет developer friendly

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+### Карточка товара
+Аналогично карточке товара из раздела "Избранное".
+### Рекомендация товара
+Аналогично рекомендации товара из раздела "Избранное".
```
и тут

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+
+# [Страница поиска](https://www.reazon.ru/search?q=%D1%82%D0%B5%D0%BB)
+### Общее
+- Все товары на странице находятся с помощью строки из поиска.
```
А надо ли все товары тестировать?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+### Описание товара и фото
+  ![item-description](https://user-images.githubusercontent.com/57019979/221987871-11bd5b29-3e96-49d5-9682-48194dae6781.png)
+  - Название характеристики и она сама помещаются целиком, их разделяют точки
+  - *Незначительный баг*. Левая часть точек отличается от правой размером и высотой расположения
```
К багам в целом стоит приложить скрины и выделить их

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+   - Кнопка изменения количества товара в корзине
+     - Если товара еще нет в корзине отобрадается надпись "В корзину"
+     - Если товар уже есть в корзине отображается количество товара в корзине и отображаются кнопки "+" и "-" для увеличения и уменьшения количества товара в корзин
+     - При первоначальном добавлении в корзину кнопка меняет вид с первого варианта на второй, при удалении последнего товара появлется первый вариант
```
`Первый` и `второй` тоже не стоит использовать, каждый пункт должен по минимуму ссылаться на другие, лучше явно раскрыть, пройдитесь по чек листу и проверьте что каждый пункт атомарен по своей сути

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+### Отзывы
+![read-review](https://user-images.githubusercontent.com/57019979/221988118-24507d47-40d2-495e-8895-e91eb23ae77d.png)
+ - Кнопка оставить отзыв
+      - Если польозватель авторизован, при клике открывается страница написания отзыва, если данный пользователь еще не оставлял отзыва на данный товар. В противном случае появляется соответсвующиее сообщение
```
Несколько пунктов в одном, стоит разбить

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+ - Кнопка оставить отзыв
+      - Если польозватель авторизован, при клике открывается страница написания отзыва, если данный пользователь еще не оставлял отзыва на данный товар. В противном случае появляется соответсвующиее сообщение
+ - Аватар
+      - Если у пользователя нет аватара, то отображается заглушка, иначе его аватар
```
Тоже несколько пунктов в одном, видел подобное и выше, пройдитесь по чек листу и проверить что один пункт - одна проверка

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+### Название товара
+![item-write-review-description](https://user-images.githubusercontent.com/57019979/221988092-e0abc056-30d1-4321-b99c-d40a190695c7.png)
+      - Полностью аналогичен блоку "Название товара" со страницы написания отзыва
+ - *Незначительный баг.* Если пользватель неавторизован, то название вкладки отображается некорректно
```
Опять же некорректно и стоит добавить ссылку на блок про который идет речь в `аналогичен`

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+### Создание отзыва
+![image](https://user-images.githubusercontent.com/57019979/222456421-d88e3910-3d91-4f14-8b2f-f6565cfdc1d8.png)
+ - Выбор рейтинга
+    - Звездочки загораются при наведении курсора, если рейтинг не выбран. Если рейтинг выбран, то данное поведение будет сохраняться только для более высокого рейтинга, чем выбран ранее. 
```
Опять же несколько в одном

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+ - Выбор рейтинга
+    - Звездочки загораются при наведении курсора, если рейтинг не выбран. Если рейтинг выбран, то данное поведение будет сохраняться только для более высокого рейтинга, чем выбран ранее. 
+    - Без выбора рейтинга нельзя отправить отзыв, будет появляться соответствующее сообщение об ошибке
+ -  Ограничений на символы нет, но есть ограничение по длине текста в каждом поле в 300 символов. Разрешается оставлять пустые поля ввода
```
Тоже несколько в одном

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+  <img src="/images/block_total.png" alt="Блок 'Итого'">
+</kbd>
+
+- Ссылка на спобод доставки
```
способ

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+- Ссылка на спобод доставки
+  - при клике страница перематывается до блока "Доставка"
+- Ссылка на дату доставки
+  - при клике страница перематывается до блока "Доставка"
```
Про перемотку в стоит объединить в один блок

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,510 @@
+  - попытка сабмита без промокода
+    - пустое поле не проходит, выводится сообщение с просьбой ввести промокод
+  - ввод недопустимых символов: \|[]{}'":;<>,~@#$%^&*+=
+    - поле не принимает символы: \|[]{}'":;<>,~@#$%^&*+= и выводится сообщение о некорректном вводе
```
опять же `некорректный`

alexey-ershkov reviewed (changes requested):
Привет) отписал комментов, главные ошибки:
- Используете много слово "корректно", а это не описывает ожидаемое поведение, это антипаттерн
- Много где несколько пунктов объединены в один, стоит разбить
- Для багов стоит докинуть скрины и не стоит разделять их по степени важности, это нужно будет делать в ДЗ2

thelvv committed:
12 fix (#6)

* initial commit

* поправил формулировки, убрал классификацию багов

Liza1040 committed:
Убрана классификация багов, исправлены замечания по странице корзины (#7)

EuphoriaAbsorber committed:
fixes for mentor's comments (#8)

* fixes for mentor's comments

* fixes for comments

tUnknownLegend committed:
[6, 7, 8, 9, 10, 11] fix (#5)

* Исправление комментов

* Соблюдение атомарности

* Добавлены линки на ссылки с аналогичными блоками

Так же исправления ссылки в содержании

* Убрал классификацию багов

* Update Base.md

* EuphoriaAbsorber comments fix

* Update Base.md

tUnknownLegend committed:
fix link in content

tUnknownLegend committed:
fix refs to favourites

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+4. [Страница поиска](https://github.com/tUnknownLegend/TP-QA-HW-1/blob/Base/Base.md#%D1%81%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B0-%D0%BF%D0%BE%D0%B8%D1%81%D0%BA%D0%B0)
+5. [Страницы авторизации](https://github.com/tUnknownLegend/TP-QA-HW-1/blob/Base/Base.md#%D0%B0%D0%B2%D1%82%D0%BE%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D1%8F)
+6. [Страница товара](https://github.com/tUnknownLegend/TP-QA-HW-1/blob/Base/Base.md#c%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B0-%D1%82%D0%BE%D0%B2%D0%B0%D1%80%D0%B0)
+7. [Страница просмотра отзывов]([https://www.reazon.ru/comment/43](https://github.com/tUnknownLegend/TP-QA-HW-1/blob/Base/Base.md#c%D1%82%D1%80%D0%B0%D0%BD%D0%B8%D1%86%D0%B0-%D0%BF%D1%80%D0%BE%D1%81%D0%BC%D0%BE%D1%82%D1%80%D0%B0-%D0%BE%D1%82%D0%B7%D1%8B%D0%B2%D0%BE%D0%B2))
```
Ссылка поломана в Markdown
<img width="498" alt="Screen Shot 2023-03-12 at 15 22 36" src="https://user-images.githubusercontent.com/56301773/224544390-c8a75f10-2fb9-4084-b9e3-51e1b91c5f92.png">

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+ - Параметры заказа: оплата, адрес, дата доставки, время доставки совпадают с указанными при создании заказа.
+ - Дата оформления заказа верно показывает число, когда заказ был оформлен.
+ - При нажатии на кнопку "Отменить заказ" статус заказа меняется с "Создан" на "Отменен", пропадает кнопка отмены.
+ - *Баг*. Одинаковый тип оплаты при создании заказа с привязанной картой и при создании заказа с выбором оплаты на месте.
```
Докиньте скринов ко всем багам

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+- *Баг*. Список товаров в избранном без применения сортировок некорректен, некоторые товары дублируются.
+  - *Корректное поведение*. Ровно по одной карточке каждого товара.
+- Сортировка товаров по рейтингу по убыванию и возрастанию работает без ошибок, товары отображаются в необходимой последовательности.
+- Сортировка товаров по цене по убыванию и возрастанию работает без ошибок, товары отображаются в необходимой последовательности.
```
Не оч понятно, что такое необходимая последовательность, плюс две проверки в одной

По хорошему стоит разбить на две проверки плюс сформулировать конкретнее
`при клике на фразу "по цене" и при отображении соотвествующей иконки происходит сортировка товаров по возрастанию цены`, больше конкретики, меньше общих слов

Посмотрите плиз везде

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+- При нажати на кнопку сортировки по цене товары сортируются по цене по убыванию и появляется иконка сортировки. При повторном нажатии товары сортируются по возрастанию и иконка сортировки крутится на 180 градусов.
+- Сортировка по рейтингу работает аналогично.
+- При перезагрузке страницы сортировка сохраняется.
+- Иконка направления сортировки правильно показывает направление сортировки.
```
А у вас иконка не съехала 
<img width="134" alt="Screen Shot 2023-03-12 at 15 27 26" src="https://user-images.githubusercontent.com/56301773/224544696-032579fc-264a-4f17-97b0-2beccd59bf20.png">

Мб это баг?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+- Рейтинг и кнопка "В корзину" работают аналогично таковым на карточке товара.
+- Если товар со скидкой, то процент скидки правильно считается из номинальной и итоговой стоимостей.
+
+# [Категории](https://www.reazon.ru/category/phones)
```
А где проверки на специфичные отображения этой страницы? Что соотвествует той же категории? А то какой смысл этой страницы, если там все то же самое что и в избранном?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+Аналогично рекомендации товара из раздела "Избранное".
+[Избранное](https://github.com/tUnknownLegend/TP-QA-HW-1/blob/Base/Base.md#%D0%B8%D0%B7%D0%B1%D1%80%D0%B0%D0%BD%D0%BD%D0%BE%D0%B5)
+
+# [Страница поиска](https://www.reazon.ru/search?q=%D1%82%D0%B5%D0%BB)
```
Докиньте тут тоже больше проверок на поисковую строку и тд, а то оч общая проверка

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+
+# [Категории](https://www.reazon.ru/category/phones)
+### Общее
+- Все товары на странице категории относятся к категории, соответствующей странице.
```
Очень общая проверка, нужна конкретика

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+    - Максимальное количество символов в поле 16.
+- При несовпадении данных пользователя выводится подсказка, что неверна почта/пароль.
+- При соблюдении всех условий при нажатии "Войти" пользователь будет авторизован и перенаправлен на главную страницу.
+- Переход на страницу регистрации по ссылке работает корректно.
```
Опять `корректно`, просил на прошлом этапе ревью пройтись и выпилить все общие слова

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+- При несовпадении ввода в полях "Пароль" и "Повторить пароль" появляется подсказка, что пароли не совпадают.
+- При попытке регистрации с уже существующей почтой возникнет ошибка.
+- При соблюдении всех условий при нажатии "Зарегистрироваться" пользователь будет зарегистрирован и перенаправлен на главную страницу.
+- Переход на страницу авторизации по ссылке работает корректно.
```
И тут

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+![image](https://user-images.githubusercontent.com/65418582/222963313-b89f62fb-ba6e-4266-822d-d9c6b3f66f52.png)
+- Инпут "Имя":
+    - При нажатии "Зарегистрироваться" в случае пустого инпута справа от него появляется подсказка, что поле должно быть заполнено.
+    - *Недоработка*. Любой непустой ввод не вызывает ошибки.
```
Баг, а не недоработка, переименуйте везде под баг

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+![item-title](https://user-images.githubusercontent.com/57019979/221987819-798f374a-6e14-4f9d-a2a5-9f158373fcf1.png)
+ - Название помещается целиком
+ - При клике на название загружается страница этого же товара
+ - Категория соответствует категории данного товара
```
Общая формулировка, распишите подробнее где и что смотреть

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+ - Название помещается целиком
+ - При клике на название загружается страница этого же товара
+ - Категория соответствует категории данного товара
+ - При клике на категорию загружаются все товары данной категории
```
Добавите проверки конкретные, как минимум переход на страницу другую должен произойти и это нужно написать

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+ -  Ограничений на символы нет, но есть ограничение по длине текста в каждом поле в 300 символов. 
+ -  Разрешается оставлять пустые поля ввода
+ -  При попытке отправить строки с целью эксплуатации XSS угрозы, отзыв оформлется, но потенциально вредностные символы в отзыв не попадают и скрипт не исполняется
+ -  При клике на кнопку "Отправить отзыв" создается отзыв и открывается страница просмотра отзывов данного товара
```
Ко всем критериям со переходом на страницу нужно добавить ссылки, посмотрите везде по чек-листу

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+    - При нажатии на саджест выполняется соответсвующий поисковой запрос
+    - Подсказка отображается, только в случае, если введенный в инпут текст похож на название товара или категории
+- Кнопка заказы
+  - При нажатии открывается страница заказов
```
Докинуть ссылок здесь и ниже

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+ 
+# Футер
+ ![image](https://user-images.githubusercontent.com/57019979/221992147-701add9b-c75d-44ad-b7bb-b4961f631e36.png)
+- Не перерендеривается
```
Как проверить? какие критерии?

alexey-ershkov commented on Base.md:
```diff
@@ -0,0 +1,541 @@
+- Кнопка "Изменить"
+  - при клике появляется попап для выбора адреса
+- Попап для выбора адреса
+  - при клике на кнопку "Добавить" открывается страница профиля
```
Добавить ссылку

alexey-ershkov reviewed (changes requested):
Привет, отписал комментов, главные моменты:
- Добавить ко всем багам скрины
- Добавить ссылки где они ожидаются
- Довыпилить общие формулировки (с корректно и пр)

когда работаете с чек листом держите в голове, что человек, который не знаком с вашим проектом смог его проверить не задав вам ни одного вопроса и однозначно сказать есть ли какие-то проблемы или нет

thelvv committed:
Фикс отчета по странице профиля (#9)

* initial commit

* initial commit

* добавил скрины для багов, убрал "недоработку"

* minor fixes

Liza1040 committed:
Добавлены ссылки (#12)

tUnknownLegend committed:
Исправление комментариев (#10)

* Исправление комментариев

- Добвлены ссылки
- Инструкция по проверке какие компоненты перерендриваются
- Другие исправления

* исправил ссылки

* Update Base.md

* Расписал категорию товара

EuphoriaAbsorber committed:
Update Base.md (#11)

* Update Base.md

* Update Base.md

* Update Base.md

alexey-ershkov reviewed (approved):
Аппрув
